doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -222,25 +222,25 @@ When the device leaves the last fabric, one of several reactions can be set to h
 
 To enable one of the reactions to the last fabric removal, set the corresponding Kconfig option to ``y``:
 
-* :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_NONE` - Do not react to the last fabric removal.
+* :ref:`CONFIG_CHIP_LAST_FABRIC_REMOVED_NONE` - Do not react to the last fabric removal.
```

Comment:
This was new to me. Before, the link needed to be in format :ref:`CONFIG_CHIP_LAST_FABRIC_REMOVED_NONE <CONFIG_CHIP_LAST_FABRIC_REMOVED_NONE>`. But apparently, not any more, because these seem to work fine.